rental.rb

```diff
@@ -0,0 +1,9 @@
+class Rental
+  attr_accessor :date
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @rentee = person
+  end
```

Comment:
- Please Make sure that the `rental belongs to a book`, Also the `rental belongs to a Person`. Kindly create their `belongs-to` relationship.
- I think there is also a typo in your code. please check `@rentee = person`
- Open the link 👉[ruby](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship%20def%20initialize(date,%20book,%20person)) for an Hint